src/main/scala/org/apache/spark/sql/kinesis/KinesisWriteTask.scala

```diff
@@ -73,14 +134,17 @@ private[kinesis] class KinesisWriteTask(producerConfiguration: Map[String, Strin
       }
 
       override def onSuccess(result: UserRecordResult): Unit = {
-        val shardId = result.getShardId
-        sentSeqNumbers = result.getSequenceNumber
+        logDebug(s"Successfully put records: \n " +
+          s"sequenceNumber=${result.getSequenceNumber}, \n" +
+          s"shardId=${result.getShardId}, \n" +
+          s"attempts=${result.getAttempts.size}")
       }
+
     }
+
     Futures.addCallback(future, kinesisCallBack)
 
     producer.flushSync()
```

Comment:
@leslieyanyan @itsvikramagr   
The slowness is on account of this function call `producer.flushSync()`.  Please refer my comment here: https://github.com/qubole/kinesis-sql/pull/81#pullrequestreview-480684080

The new code in this PR is showing improved performance because method `sendBundledData()` doesn't have this function call `producer.flushSync()`

We'll need to separately evaluate how much performance impact we're getting by using `GroupedIterator` instead of normal iterator.